packages/migrate/src/commands/DbPull.ts

```diff
@@ -338,6 +340,32 @@ Learn more about the upgrade process in the docs:\n${link(
         console.error(introspectionWarningsMessage.replace(/(\n)/gm, '\n// '))
       }
     } else {
+      if (schemaPath) {
+        const schema = await getSchema(args['--schema'])
+        const config = await getConfig({
```

Comment:
Not something to do in the very short term, but we should rethink our `getConfig` usage in introspection/migrate at some point.